eZ/Publish/Core/Persistence/Cache/LocationHandler.php

```diff
@@ -63,6 +63,17 @@ public function loadSubtreeIds($locationId)
         return $locationIds;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
+     */
+    public function loadAllNodes()
```

Comment:
What is a "Node"? ;)

Also shouldn't this be paged and return Locations? 

And if you only want raw data in database, then use database connection in command. Otherwise we are bleeding schema details up to other layers.